File: launchdarkly/ruby/src/CreateOAuth2ClientExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

oauth_client_post = LaunchDarklyClient::OauthClientPost.new

begin
    response = LaunchDarklyClient::OAuth2ClientsApi.new.create_o_auth2_client(
        oauth_client_post,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling OAuth2ClientsApi#create_o_auth2_client: #{e}"
end

File: launchdarkly/python/src/CreateOAuth2ClientExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    oauth_client_post = models.OauthClientPost(
        name=None,
        redirectUri=None,
        description=None,
    )

    try:
        response = api.OAuth2ClientsApi(api_client).create_o_auth2_client(
            oauth_client_post=oauth_client_post,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling OAuth2ClientsApi#create_o_auth2_client: %s\n" % e)
